cypress/utils/utils.ts

```diff
@@ -504,6 +538,44 @@ export function verifySortDesc(listToVerify: Array<any>, unsortedList: Array<any
     });
 }
 
+export function verifyDateSortAsc(listToVerify: Array<string>, unsortedList: Array<string>): void {
```

Comment:
**nitpick:** This can be simplified with

`listToVerify: string[], unsortedList: string[])`

Same comment for the `verifyDateSortDesc` method

But that's a matter of personal taste.